static_html/projects_page.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html>
+<head>
+  <title>Lindsey's Projects</title>
+</head>
+
+<body>
+
+<header>
+  <h1>Lindsey's Website</h1>
+  <nav>
+    <a href="/">Home</a>
+    <a href="about">About</a>
+    <a href="blog">Blog</a>
+  </nav>
+</header>
+<br>
+
+<h2>Projects</h2>
+<br>
+
+<article>
+  <h3>Calculator</h3>
+    <p>My thoughts on the Calculator project</p>
+    <a href="https://github.com/lindseyreno/C3Projects--Calculator/blob/lar/master/calculator.rb">Calculator Code</a>
+</article>
+<br>
+
+<article>
+  <h3>Random Menu Generator</h3>
+    <p>My thoughts on the Random Menu Generator</p>
+    <a href="https://github.com/lindseyreno/C3Projects--RandomMenuGenerator/blob/lar/master/random_menu.rb">Random Menu Generator Code</a>
+</article>
+<br>
+
+<article>
+  <h3>Scrabble</h3>
+  <p>My thoughts on Scrabble</p>
+  <a href="https://github.com/lindseyreno/C3Projects--Scrabble/blob/lar/master/lib/scrabble.rb">Scrabble Code</a>
+  <a href="https://github.com/lindseyreno/C3Projects--Scrabble/blob/lar/master/spec/scrabble_spec.rb">Scrabble Spec Code</a>
+</article>
+<br>
+
+<h4>Check out the rest of my code on github!</h4>
+<a href="https://github.com/lindseyreno"><img src="images/github_image.png" height="45" width="45"></a>
```

Comment:
There's no content between the `<a>` and `</a>` so I don't have anything to click on. :(